sdk/purview/Azure.Analytics.Purview.Workflows/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-10-26)
 
 ### Features Added
 
+- Add `/workflows/{id}/validate` API.
+- Add admin query options to workflow / workflow run APIs.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- Split `PurviewWorkflowServiceClient` into multiple clients by model.
 
 ### Other Changes
 
+- Update descriptions of some APIs.
```

Comment:
This doesn't provide any value to customers reading the changelog; let's remove this.

Reply:
Thanks. Removed this.